html/js/tree/Move.js

```diff
@@ -20,9 +20,12 @@ GTE.TREE = (function (parentModule) {
         return "Move: " + "parent: " + this.parent + "; child: " + this.child;
     };
 
+    /**
+    * Draws the move (a line) in the canvas
```

Comment:
Please leave a TODO comment here reminding that this function will need to be rewritten once we have >1 nodes per iset.

Maybe you can reuse an existing Issue (whatever you prefer)